src/main/java/uk/gov/companieshouse/config/FeatureToggleConfig.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FeatureToggleConfig {
+
+    @Value("${featureToggles.automaticallyRequestRefund}")
```

Comment:
bit of a mouthful - how abouts just `featureToggles.refunds` and `isRefundsEnabled`?